src/RpcServer/RpcServer.Node.cs

```diff
@@ -73,23 +73,31 @@ protected virtual JToken GetVersion(JArray _params)
             json["nonce"] = LocalNode.Nonce;
             json["useragent"] = LocalNode.UserAgent;
             json["protocol"] = new JObject();
-            json["protocol"]["addressversion"] = system.Settings.AddressVersion;
-            json["protocol"]["network"] = system.Settings.Network;
-            json["protocol"]["validatorscount"] = system.Settings.ValidatorsCount;
-            json["protocol"]["msperblock"] = system.Settings.MillisecondsPerBlock;
-            json["protocol"]["maxtraceableblocks"] = system.Settings.MaxTraceableBlocks;
-            json["protocol"]["maxvaliduntilblockincrement"] = system.Settings.MaxValidUntilBlockIncrement;
-            json["protocol"]["maxtransactionsperblock"] = system.Settings.MaxTransactionsPerBlock;
-            json["protocol"]["memorypoolmaxtransactions"] = system.Settings.MemoryPoolMaxTransactions;
-            json["protocol"]["initialgasdistribution"] = system.Settings.InitialGasDistribution;
-            json["protocol"]["hardforks"] = new JArray(system.Settings.Hardforks.Select(hf =>
+            // rpc settings
+            JObject rpc = new();
+            rpc["maxiteratorresultitems"] = settings.MaxIteratorResultItems;
+            rpc["sessionenabled"] = settings.SessionEnabled;
+            // protocol settings
+            JObject protocol = new();
+            protocol["addressversion"] = system.Settings.AddressVersion;
+            protocol["network"] = system.Settings.Network;
+            protocol["validatorscount"] = system.Settings.ValidatorsCount;
+            protocol["msperblock"] = system.Settings.MillisecondsPerBlock;
+            protocol["maxtraceableblocks"] = system.Settings.MaxTraceableBlocks;
+            protocol["maxvaliduntilblockincrement"] = system.Settings.MaxValidUntilBlockIncrement;
+            protocol["maxtransactionsperblock"] = system.Settings.MaxTransactionsPerBlock;
+            protocol["memorypoolmaxtransactions"] = system.Settings.MemoryPoolMaxTransactions;
+            protocol["initialgasdistribution"] = system.Settings.InitialGasDistribution;
+            protocol["hardforks"] = new JArray(system.Settings.Hardforks.Select(hf =>
             {
                 JObject forkJson = new();
                 // Strip "HF_" prefix.
                 forkJson["name"] = StripPrefix(hf.Key.ToString(), "HF_");
                 forkJson["blockheight"] = hf.Value;
                 return forkJson;
             }));
+            protocol["rpc"] = rpc;
```

Comment:
```suggestion
            json["rpc"] = rpc;
```
Is this better?

Reply:
We want protocol and rpc or rpc inaide the protocol? @roman-khimov

Reply:
Can be any way, but I think @superboyiii suggestion is slightly better, leaving `protocol` for core things (that are the same per-network) and then `rpc` is just some kind of presentation, not a part of it. And it can vary from node to node.